docs/DEVGUIDE.md

```diff
@@ -18,6 +28,22 @@ layout:
     ├── pkg            # Contains all non-"main" Go packages
     └── vendor         # Glide-managed dependencies (untracked)
 
+## Working on Issues
+
+Github does not allow non-maintainers to assign, or be assigned to, issues.
```

Comment:
not sure... I'm leaning towards "yes" so that everyone follows the same process.  I stole this idea from Docker and I think they allow both -which means people have to look for both. I haven't heard people complain yet about that duplication.